cmd/main.go

```diff
@@ -94,6 +94,8 @@ var (
 func main() {
 
 	log.SetOutput(os.Stderr)
+	log.SetPrefix("NOTE: ")
+	log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
```

Comment:
what're these doing? Do they affect all log statements?

Reply:
They do affect all log statements.

SetPrefix just adds that prefix to all log lines. 

SetFlags removes the timestamp from the log. Since this is for the CLI output and it's a short run CLI, I don't think the timestamps make sense and I think the "NOTE: " prefix makes it more human readable. Although, if you have a better prefix suggestion, I'd be interested.

Reply:
ah no I was worried this affected logs when consumed as a library. this totally makes sense for the cli case